execution_engine/src/core/tracking_copy/mod.rs

```diff
@@ -329,9 +322,7 @@ impl<R: StateReader<Key, StoredValue>> TrackingCopy<R> {
     pub fn write(&mut self, key: Key, value: StoredValue) {
         let normalized_key = key.normalize();
         self.cache.insert_write(normalized_key, value.clone());
-        self.ops.insert_add(normalized_key, Op::Write);
-        let transform = Transform::Write(value);
-        self.journal.push((normalized_key, transform));
```

Comment:
Is this correct? If we create a `Write` transform then we should also add it to the `ops` collection.

Reply:
I see - `ops` collection is removed entirely.